<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Capacitor</class>
 <widget class="QDialog" name="Capacitor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>456</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Capacitor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="horizontalSpacing">
      <number>65</number>
     </property>
     <property name="verticalSpacing">
      <number>16</number>
     </property>
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="capacitor_Name_label">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QLabel" name="capacitor_Alias_label">
       <property name="text">
        <string>Alias:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="capacitor_Id_label">
       <property name="text">
        <string>UID:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="capacitorID_lineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>130</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhNone</set>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
       <property name="dragEnabled">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="capacitorName_lineEdit">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="capacitorAlias_lineEdit">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>Rated Voltage [kV]:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="capacitorRatedVoltage_comboBox">
       <property name="editable">
        <bool>false</bool>
       </property>
       <property name="currentText">
        <string>0.23</string>
       </property>
       <item>
        <property name="text">
         <string>0.23</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>0.40</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>0.42</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>0.52</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3.00</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6.00</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6.30</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10.00</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10.38</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10.50</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>11.00</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>13.00</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>20.00</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>20.50</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>33.00</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>35.00</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>36.75</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>66.00</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>110.00</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>132.00</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>220.00</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>400.00</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Maximum Voltage [kV]:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="capacitorMaxVoltage_lineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_19">
       <property name="text">
        <string>Number Of Phases:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="capacitorNumberOfPhasescomboBox">
       <property name="duplicatesEnabled">
        <bool>false</bool>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>Single Phase</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Two Phases</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Three Phases</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Minumum Voltage [kV:]</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="capacitorMinVoltage_lineEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Connection Type:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="capacitorConnectionType_comboBox">
       <item>
        <property name="text">
         <string>Delta</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Wye</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Autotransformer</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Wye with ground</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Zigzag</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Nominal Section  [1]:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLineEdit" name="capacitorNominalSection_lineEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Description:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="4">
      <widget class="QPlainTextEdit" name="capacitorDescription_plainTextEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Reactive power per section</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Reactive Power [kVAr]:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="capacitorReactivePower_lineEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Active Power Loss [kW:]</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="capacitorActivePowerLoss_lineEdit"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="capacitor_buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Discard|QDialogButtonBox::Save</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
